src/lib/network/clients/save_network.rb

```diff
@@ -220,6 +204,33 @@ def copy_from_instsys
       nil
     end
 
+    WICKED_DHCP_PATH = "/var/lib/wicked/".freeze
+    WICKED_DHCP_FILES = ["duid.xml", "iaid.xml", "lease*.xml"].freeze
+    DHCP_CLIENT_PATH = "/var/lib/dhcpcd/".freeze
+    DHCP_CLIENT_CACHE = "dhcpcd-*.cache".freeze
+    DHCPv6_CLIENT_CACHE_PATH = "/var/lib/dhcpv6".freeze
+
+    def copy_dhcp_info
+      # Ensure directories exist
+      wicked_dest_dir = Installation.destdir + WICKED_DHCP_PATH
```

Comment:
We usually use http://ruby-doc.org/core-2.5.0/File.html#method-c-join or similar

Reply:
true :+1: